pages/spicedb/concepts/datastores.mdx

```diff
@@ -30,6 +30,44 @@ spicedb migrate head \
 
 For more information, refer to the [Datastore Migration documentation](/spicedb/concepts/datastore-migrations).
 
+## Read replicas for Postgres and MySQL
+
+SpiceDB can reference read-only replicas of permissions data stored in Postgres and MySQL that respect consistency requirements by detecting if a given revision has been replicated from the
```

Comment:
I'm not sure `reference` is the right term here. I'd perhaps use `make use` or `read from`.

Reply:
`s/that respect consistency requirements/,and maintain consistency guarantees`

Reply:
changed!